### ISSUE TITLE
Is 6.11.4 an official release?

### ISSUE THREAD

**Issue by rwlodarczyk-xealth** [protobufjs/protobuf.js#1928]
protobuf.js version: 6.11.4

NPM Registry indicates that there's a v6.11.4 release, but it's not listed under releases or in the CHANGELOG.md. Is this an official release?

**Comment by alexander-fenster** [comment]
Yes, it's official, released from [this branch](https://github.com/protobufjs/protobuf.js/tree/release-6.11.4).

6.11.4 is a backport of the recent security fixes that was needed for users that cannot update to 7.x yet. My bad that I did not include it in `CHANGELOG.md`: it did not go through the regular release automation. I actually noticed there are several more 6.x releases missing from the change log, I might find some time this week to backfill them, thank you for pointing that out.

**Comment by rwlodarczyk-xealth** [comment]
Great, thank you!
